apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/ReliableStreamReader.cs

```diff
@@ -34,7 +34,6 @@ namespace Google.Cloud.Spanner.V1 {
     internal sealed class ReliableStreamReader : IDisposable
     {
         private AsyncServerStreamingCall<PartialResultSet> _currentCall;
-        private readonly object _sync = new object();
```

Comment:
Not sure which PR this is meant to be in, but if there's no particularly logic place, I don't mind too much.